### ISSUE TITLE
Metadata is not compatible 

### ISSUE THREAD

**Issue by elmira-amirloo** [nrhine1/precog#5]
Hi Nick,

Thanks for publishing your code. The data uploaded [here](https://github.com/nrhine1/precog_carla_dataset) is different from what this version of the code expects. Can you update the uploaded data? Thanks in advance,

Cheers,

**Comment by noahcao** [comment]
Yes. I also encountered the same problem when trying to train on Carla data... And as I'm not familiar with the codebase style you are publishing in, it's hard for me to customize the code to run it successfully...
